src/ripple/app/consensus/RCLConsensus.cpp

```diff
@@ -843,7 +843,8 @@ RCLConsensus::validate(RCLCxLedger const& ledger, bool proposing)
     v->setTrusted();
     // suppress it if we receive it - FIXME: wrong suppression
     app_.getHashRouter().addSuppression(signingHash);
-    app_.getValidations().addValidation(v, "local");
+    if (! app_.getValidations().addValidation(v, "local"))
```

Comment:
Are you suggesting we only need to prevent ourselves from issuing the in the past validations, since that would usually prevent receiving them from others?  If so, I like the idea. You could eliminate the check inside `addValidation` and instead just check when considering whether to validate the consensus ledger https://github.com/mtrippled/rippled/blob/22536534936b9508f8b4dfc54d5abae869579e8e/src/ripple/app/consensus/RCLConsensus.cpp#L438